graph_csv.py

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Copyright (C) 2017 Unicamp-OpenPower
+Licensed under the Apache License, Version 2.0 (the “License”);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an “AS IS” BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import argparse
+import thread
+import time
+import os
+
+INTERVAL = 10
+
+#function used to build the ipmi and csv commands
+def build_commands(args):
+    year = time.strftime("%Y")
+    month = int(time.strftime("%m"))
+    # DO NOT FORGET TO CHANGE THIS AFTER TESTING
+    day = int(time.strftime("%d")) + 1
+    date=str(year)+str(month)+str(day)
+    
+    csv_command = 'python2.7 csvcreator.py --name=last --date='+date+' --jsonfile='
```

Comment:
As some people use python as an alias for python3, I think that's not a good idea.